test/install/Install.cpp

```diff
@@ -7,9 +7,10 @@ int main()
 {
     // Audio
     [[maybe_unused]] sf::InputSoundFile inputSoundFile;
-    [[maybe_unused]] sf::Listener       listener;
-    [[maybe_unused]] sf::Music          music;
-    [[maybe_unused]] sf::Sound          sound;
+    // TODO: Should I change this to another test like "auto volume{sf::Listen::getGlobalVolume()};" or remove it?
+    // [[maybe_unused]] sf::Listener       listener;
```

Comment:
Funny how I accidentally used one of these pseudo-classes. Just replace `listener` with an instance of some other Audio module type.

Reply:
I feel validated with this PR as a result.